cli/sawtooth_cli/submit.py

```diff
@@ -43,7 +43,14 @@ def do_submit(args):
     start = time.time()
 
     for batch_list in _split_batch_list(args, batches):
-        rest_client.send_batches(batch_list)
+        response = rest_client.send_batches(batch_list, args.wait)
+
+        # If told to wait, but timed out, print out statuses of batches
```

Comment:
How do we know if we received a full response or timed out here? Should we print some kind of notification if we timed out?